mne/viz/_brain/_brain.py

```diff
@@ -267,6 +267,12 @@ def __init__(self, subject_id, hemi, surf, title=None,
                                        bgcolor=background,
                                        shape=shape,
                                        fig=figure)
+
+        if _get_3d_backend() == "pyvista":
+            self.plotter = self._renderer.plotter
```

Comment:
Now that `Brain` is public we should probably make as many attributes as possible private, so `_plotter` is better here